dist/main.js

```diff
@@ -1,2 +1,2 @@
-"use strict";Object.defineProperty(exports,"__esModule",{value:!0}),require("atom");const e=require("./main-3af7aad3.js");require("path"),exports.Minimap=e.Minimap,exports.activate=e.activate,exports.activatePlugin=e.activatePlugin,exports.config=e.config,exports.deactivate=e.deactivate,exports.deactivateAllPlugins=e.deactivateAllPlugins,exports.deactivatePlugin=e.deactivatePlugin,Object.defineProperty(exports,"domStylesReader",{enumerable:!0,get:()=>e.domStylesReader}),Object.defineProperty(exports,"editorsMinimaps",{enumerable:!0,get:()=>e.editorsMinimaps}),exports.emitter=e.emitter,exports.getActiveMinimap=e.getActiveMinimap,exports.getConfigSchema=e.getConfigSchema,exports.getPluginsOrder=e.getPluginsOrder,exports.minimapClass=e.minimapClass,exports.minimapForEditor=e.minimapForEditor,exports.minimapForEditorElement=e.minimapForEditorElement,exports.minimapViewProvider=e.minimapViewProvider,exports.observeMinimaps=e.observeMinimaps,exports.onDidActivate=e.onDidActivate,exports.onDidActivatePlugin=e.onDidActivatePlugin,exports.onDidAddPlugin=e.onDidAddPlugin,exports.onDidChangePluginOrder=e.onDidChangePluginOrder,exports.onDidCreateMinimap=e.onDidCreateMinimap,exports.onDidDeactivate=e.onDidDeactivate,exports.onDidDeactivatePlugin=e.onDidDeactivatePlugin,exports.onDidRemovePlugin=e.onDidRemovePlugin,exports.plugins=e.plugins,exports.provideMinimapServiceV1=e.provideMinimapServiceV1,exports.registerPlugin=e.registerPlugin,exports.standAloneMinimapForEditor=e.standAloneMinimapForEditor,exports.toggle=e.toggle,exports.togglePluginActivation=e.togglePluginActivation,exports.unregisterPlugin=e.unregisterPlugin;
```

Comment:
Please do not check in the built file. GitHub Action does it automatically.

Reply:
I thought they were added to `.gitignore`?

Reply:
Yeah, but build-commit does some git magic to add them to the repo. If you run the `npm run build-commit` from bash, this should not happen. I have not found a general solution yet.

Reply:
I'm confused, I thought `npm run build-commit` commits all of the files. You are telling me now that it prevents committing the `dist` folder?

Reply:
gitignore prevents doing that, however, build-commit renames gitignore temporarily, commits the dist folder, and renames back the gitignore to what it was. Then it runs a git script to ignore the changes of dist folder. 

https://github.com/aminya/build-commit/blob/07f9fbfdd69409864b1b15319ee31a1e74f01a3e/bin/build-commit#L60

Reply:
I'm thinking just committing the `dist` folder would be easier. I don't see any downsides to it.

Reply:
build-commit works for now. Ideally, we should fix the issue in apm. In the previous community meeting, there were suggestions for using Github packages for publishing using apm.
https://github.com/atom/apm/issues/498

Reply:
That seems unlikely to happen any time soon, or at all, since it would require adding every Atom package to GitHub packages. That would mean Atom would lose any package that is not currently maintained.

I guess we will just keep having these conversations every time I forget to use build-commit 😁